### ISSUE TITLE
Misleading information on Viewer Role, Power BI

### ISSUE THREAD

**Issue by AUT-sjho** [MicrosoftDocs/feedback#3383]
**Describe the bug**
The information written on the page for viewer role in Power BI is misleading.
Even if you have a premium subscription and a premium workspace setup, the users that are assigned as viewers are prompted to upgrade to pro in order to actually view the content.

**To Reproduce**
1. Go to https://docs.microsoft.com/en-us/business-applications-release-notes/april19/business-intelligence/power-bi-service/viewer-role

2. The description about viewer role on this page is wrong and misleading.

**Expected behavior**
Be clear on what the premium workspace and viewer role provides. 
Customers will purchase premium thinking this would solve licensing issues whereas in reality it doesn't.

**Screenshots**
![image](https://user-images.githubusercontent.com/26944982/104510387-fa7f8b80-564f-11eb-9b82-a7da38c85663.png)

**Comment by gewarren** [comment]
@maggiesMSFT Can you look at this issue?

**Comment by maggiesMSFT** [comment]
Hi, @AUT-sjho -- Thanks for asking! That certainly was the way the product was working. I'll check to see if something has changed.

**Comment by AUT-sjho** [comment]
@maggiesMSFT no, definitely new workspaces, you could have a look at Case ID 120072624000614 in Power BI support for more refrence to the misleading information. I thought the documentation would have been changed by now though. We initially thought that viewers would be able to view all shared content on all platforms(web, app, desktop app) but that wasn't the case and the conclusion was that this was not an error and it was by design.

Hope the case helps clarify the errors

**Comment by maggiesMSFT** [comment]
Hi, @AUT-sjho -- here's what I've found out so far. The docs are correct as is, so:
"Sounds like a bug if it really is a user who has viewer role and not any other role. Remember that users may get roles through security groups."
I'll let you know if I hear more.
Thanks!
